src/commands/apps-sdkconfig.ts

```diff
@@ -1,56 +1,24 @@
-import * as ora from "ora";
 import * as fs from "fs-extra";
 
 import { Command } from "../command";
 import {
-  AppConfigurationData,
-  AppMetadata,
+  AppConfig,
   AppPlatform,
-  getAppConfig,
   getAppConfigFile,
   getAppPlatform,
-  listFirebaseApps,
+  getPlatform,
+  getSdkConfig,
+  getSdkOutputPath,
+  sdkInit,
+  writeConfigToFile,
 } from "../management/apps";
-import { needProjectId } from "../projectUtils";
-import { getOrPromptProject } from "../management/projects";
-import { FirebaseError } from "../error";
 import { requireAuth } from "../requireAuth";
 import { logger } from "../logger";
-import { promptOnce } from "../prompt";
 import { Options } from "../options";
+import * as path from "path";
```

Comment:
Nit: prefer grouping this with the other external imports at the top of the file.